src/memory.cpp

```diff
@@ -47,6 +47,7 @@ struct obj_data *memoryClass::GetObject()
 struct veh_data *memoryClass::GetVehicle()
 {
   struct veh_data *temp = new veh_data;
+  int veh_destruction_timer
```

Comment:
missing semicolon, this doesn't compile does it? Also, this isn't the right place to define new data for the vehicle structure, you want structs.hpp

Reply:
Does not compile, no. Thank you for the help.